handbook/business-operations/README.md

```diff
@@ -320,6 +320,40 @@ Within 60 days of the end of the year, follow these steps:
 - Make the cancellation notification in accordance with the contract terms between Fleet and the vendor, typically these notifications are made via email and may have a specific address that notice must be sent to. If the vendor has an autorenew contract with Fleet there will often be a window of time in which Fleet can cancel, if notification is made after this time period Fleet may be obligated to pay for the subsequent year even if we don't use the vendor during the next contract term.  
 - Once cancelled, update the recurring expenses section of [The Numbers](https://docs.google.com/spreadsheets/d/1X-brkmUK7_Rgp7aq42drNcUg8ZipzEiS153uKZSabWc/edit#gid=2112277278) to reflect the cancellation by changing the projected monthly burn in column G to $0 and adding "CANCELLED" in front of the vendor's name in column C.
 
+### Review an NDA
+We need to review an NDA anytime a vendor, customer or other party wants to:
+1. Use their own NDA rather than Fleet's standard NDA, or
```

Comment:
```suggestion
- Use their own NDA rather than Fleet's standard NDA, or
```
Let's use bullets here rather than numbers since it's a list rather than a sequence of steps.